backend/fms_core/filters.py

```diff
@@ -175,10 +177,16 @@ class Meta:
 
 class DatasetFilter(GenericFilter):
     release_flag = django_filters.NumberFilter(method="release_flag_filter")
+    latest_release_update = django_filters.CharFilter(method="latest_release_update_filter")
 
     def release_flag_filter(self, queryset, name, value):
         return queryset.filter(release_flag=value)
 
+    def latest_release_update_filter(self, queryset, name, value):
+        return queryset.annotate(
+            latest_release_update=Max("readsets__release_status_timestamp")
+        ).filter(latest_release_update__gte=value)
```

Comment:
if the time format of value is not the same as the one on freezeman this could cause issues but it is acceptable on our side ... Might be safer for the API to keep asking for a given period of time from the moment of the request ...